server/controllers/answer.js

```diff
@@ -105,46 +110,50 @@ class Answer {
 
     const response = voteRepo.createVote(userId, answerId, voteStatus);
     const votedAnswer = answerRepo.getAnswer(answerId);
-    switch (response) {
-      case 'downvote error':
-        return res.status(400).json({
-          status: 'error',
-          message: 'this answer has been previously downvoted by you',
-        });
-      case 'upvote error':
-        return res.status(400).json({
-          status: 'error',
-          message: 'this answer has been previously upvoted by you',
-        });
-      case 'downvote success':
-        return res.status(200).json({
-          status: 'success',
-          message: 'you have downvoted this answer',
-          votedAnswer,
-        });
-      case 'upvote success':
-        return res.status(200).json({
-          status: 'success',
-          message: 'you have upvoted this answer',
-          votedAnswer,
-        });
-      case 0:
-        return res.status(201).json({
-          status: 'success',
-          message: 'you have downvoted this answer',
-          votedAnswer,
-        });
-      case 1:
-        return res.status(201).json({
-          status: 'success',
-          message: 'you have upvoted this answer',
-          votedAnswer,
-        });
-      default:
-        return res.status(400).json({
-          status: 'error',
-          message: 'i am hoping we never get here',
-        });
+    if (response === 'downvote error') {
```

Comment:
So like the feedback I gave earlier, you can knock out multiple lines of code by just making a string check. So, for instance, we could re-write the error checks like so:

```javascript
if (response.split(" ")[1] === "error") {
    return res.status(400).json({
      status: 'error',
      message: `This answer has previously been ${response.split(" ")[0]}d by you`
  })
}
```
You could do the same for success also. Let me know if that's clear